swagger/v1/component_schemas/ListFilter.yml

```diff
@@ -5,7 +5,3 @@ properties:
     description: "Return only records that have been updated since this date and time (ISO 8601 date format)"
     type: :string
     example: "2021-05-13T11:21:55Z"
-  participant_id:
```

Comment:
We removed `participant_id` here but `GET /api/v1/participant-declarations` can actually filter by it. Do you need another component schema for this one specifically?

Reply:
I do, that's what `swagger/v1/component_schemas/ListFilterDeclarations.yml` is for :)